androidgen/src/main/java/com/azure/autorest/android/Androidgen.java

```diff
@@ -71,7 +74,9 @@ protected NodeTuple representJavaBeanProperty(Object javaBean, Property property
                 }
             };
 
-            Yaml newYaml  = new Yaml(representer);
+            LoaderOptions loaderOptions = new LoaderOptions();
```

Comment:
Can this be refactored to share code with all other *gen libraries so we don't have to duplicate this in 3 places?

Reply:
Alternative is to put it in extension-base, but seems not matter too much.

And there might arise some circumstance that one plugin really want to limit the max number (though not very likely, as this is only to consume output from m4, not some data from user).